agent/src/runtime_connectors/podman_cli.rs

```diff
@@ -382,6 +473,8 @@ mod tests {
             state: PodmanContainerState::Created,
             exit_code: 0,
             labels: Default::default(),
+            pod: "".into(),
+            id: "".into(),
```

Comment:
We could have added a generate_test_podman_container_info function that takes a state and an exit code and returns an object with default values for the rest of the attributes.